src/scripts.js

```diff
@@ -22,7 +21,10 @@ const filterBar = document.querySelector('.filter-section-js');
 const mainView = document.querySelector('.main-view-container-js');
 const searchRecipesButton = document.getElementById('searchRecipes');
 const filterButton = document.getElementById('filterButton');
-// const antipasti = document.getElementById('antipasti')
+const homeButton = document.querySelector('.home-button');
```

Comment:
small detail, but for the sake of being consistent across the board let's make sure to add "-js" to any class name that is being used in our scripts file